gridt/src/app/core/api.service.ts

```diff
@@ -267,7 +269,7 @@ export class ApiService {
 
     return this.auth.readyAuthentication$.pipe(
       flatMap((options) => this.http.post<ServerMessage>(
-        `${this.URL}/change_password`, {old_password: old_password, new_password: new_password}, options
+        `${this.URL}/user/change_password`, {old_password: old_password, new_password: new_password}, options
```

Comment:
Fun fact, there is syntactic sugar here, you can write 
```ts
{ old_password, new_password }
```
and it should evaluate to the same thing.